ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -297,6 +296,11 @@
      * The key for the {@link RequestPath} that represents the parsed path of an incoming request.
      */
     public static final String REQUEST_PATH = KEY_PREFIX + "request-path";
+
+    /**
+     * To be set to {@code true} if failures reason should be attached to frontend responses.
+     */
+    public static final String SEND_FAILURE_REASON = KEY_PREFIX + "send-failure-reason";
```

Comment:
thanks for making this change! I can change some of the code inside of AmbryLI to use this instead of the old approach.